scm/doc/TechGuide/chap_hsd.tex

```diff
@@ -0,0 +1,212 @@
+\chapter{Hierarchical Physics Development}
+\label{chapter: Hierarchical_Physics_Development}
+
+Chapter 7 of the CCPP v6 Technical Documentation (\url{https://ccpp-techdoc.readthedocs.io/en/v6.0.0/}) provides an overview of the tools supported by the SCM to faciliate hierarchical systems systems development (HSD)
+
+\section{Background}
+
+Developing and implementing a new physics parameterization for use in an operational setting requires extensive testing and evaluation. This is to ensure that new developments aren’t yielding unexpected results and that all computational considerations are being met. From a scientific perspective, this process should be incremental and hierarchical: starting with initial testing of a simple idealized case, then progressing to fully-coupled high-resolution global forecasts on high-performance computing systems.
+
+
+The CCPP-SCM provides developers working within CCPP compliant host models the ability to test their physics innovations without having to worry about the coupling to the dynamical core. This is a critical step in the model development hierarchy, providing insight on how an introduced physics change can modify the evolution of the internal physics state. However, there are still challenges, most notably the interactions between introduced changes and the other physics schemes in the suite.
+
+\section{CCPP Suite Simulator}
+
+\subsection{Overview}
+
+The CCPP Suite Simulator is a CCPP compliant physics scheme that provides the ability to turn on/off physical processes in a Suite Definition File (SDF), using namelist options. This simulator `piggybacks' on an existing SDF, replacing physics tendencies with data-driven tendencies (\ref{fig:CSS_tendency_schematic}). 
+
+\begin{figure}[h]
+    \centering
+    \includegraphics[width=0.80\textwidth]{images/CSS_tendency_schematic.png}
+    \caption{Equation for internal physics state evolution for process-split physics suite, where S is the prognostic state and D are simulated data tendencies. Top) Standard Suite Definition File; Middle) Active PBL physics with simulated tendencies for other schemes; Bottom) Active PBL and radiation, with simulated tendencies for other schemes.}
+    \label{fig:CSS_tendency_schematic}
+\end{figure}
+
+\subsection{Process-split vs. Time-split Physics Process}
+
+Process-split physics processes are schemes that rely on the state provided the model's dynamical core, whereas time-split processes use the state provided by the previous physics process. A SDF can be any combination of time-split and process-split schemes, just as long as the appropriate interstitial schemes are created to couple the physics schemes.
```

Comment:
Process-split physics processes are schemes that rely on the state provided **(by)** the model's dynamical core

Reply:
I would also say that "process-split physics processes are schemes that share a common input state, whereas ..". The state doesn't have to come from the dycore. You could imagine a scenario where you have some time-split (or even more process-split) schemes called before a set of process-split schemes.